website/static/css/style.css

```diff
@@ -666,4 +666,8 @@ button.close {
 
 .comma-separated:last-of-type:after {
     content: "";
-}
\ No newline at end of file
+}
+
+#render-node .ball-pulse-sync>div, #render-node .ball-pulse>div, #render-node .ball-scale>div, #render-node .ball-beat-sync>div, #render-node .ball-beat>div, #render-node .ball-beat>div {
```

Comment:
Yes, the ball-scale loader should be usable everywhere; I am referring more to this selector--it's seems overly specific, due mostly to the fact that it's using a CSS ID.

I see now that this is a copy-and-paste of https://github.com/CenterForOpenScience/osf.io/blob/45caed51a62ec0cebafa36202b78f34796230995/website/static/css/my-projects.css#L329-L331 . Instead of relying on IDs, can you make a new class name that describes its usage (something like `.ball-scale-dark`, `.ball-scale-light`...)